NEWS.md

```diff
@@ -1,3 +1,6 @@
+## 19.0.1 2020-06-16
+* Service points with many locations do not allow pick slips to be printed (CIRC-768)
```

Comment:
I usually leave a gap between the heading and the description of the contents.

Reply:
That's right. Sorry I missed that. Should I recreate this PR with correct formatting?

Reply:
Good question. 

No, I don't think it's worth re-doing the release. 

Maybe just push a new commit to change the formatting for the future. And change the formatting in the GitHub release notes.

Reply:
Pushed a fix. What do you mean by changing the formatting in the release notes? Theres is just [one line in the description](https://github.com/folio-org/mod-circulation/releases). I also created an identical PR for Flameflower: https://github.com/folio-org/mod-circulation/pull/574

Reply:
That's fine, there is no need to change that.

Reply:
Thank you